singlecellmultiomics/bamProcessing/bamToCountTable.py

```diff
@@ -578,6 +578,11 @@ def create_count_table(args, return_df=False):
     if return_df:
         return df
 
+    if args.bulk is True:
```

Comment:
This statement probably needs to be done before line `578`?  Otherwise passing this flag does not have effect when using the `return_df` option

Reply:
I didn't do that as I felt it would probably break something. But I tried it, and even though it runs on my local machine, the pytest check fails giving errors regarding missing attributes. See commit: 3c7bb7f9862adf0a4e936025c4a8524b81d03311. So I'm not quite sure yet what the issue is there, but if I place the statements after 579 all works and pytest is happy. =/